addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -42,7 +42,18 @@
                         </t>
                     </t>
                 </div>
-                <div class="o_ThreadViewTopbar_actions d-flex align-items-center">
+                <div class="o_ThreadViewTopbar_actions d-flex align-items-center ml-1">
+                    <t t-if="messaging.currentGuest and !messaging.device.isMobile">
```

Comment:
I don't think we want this in the backend, (extra) condition should probably be on threadview(er) being related to discuss public view.